src/js/common/schemaform/FormPage.jsx

```diff
@@ -102,6 +104,15 @@ class FormPage extends React.Component {
     return { pages: expandedPageList, pageIndex };
   }
 
+  autoSave() {
+    const { form } = this.props;
+    const data = form.data;
+    const { formId, version } = form;
+    const returnUrl = this.props.locationPathname;
+
+    this.props.saveInProgressForm(formId, version, returnUrl, data, true);
```

Comment:
I think we want to avoid auto-saving if they're not logged in, so we don't show them that red box the whole time they're filling out the form if they're not going to log in.

Reply:
Good idea, `SaveStatus` is now rendered if `this.props.isLoggedIn` is true.